app/angular/package.json

```diff
@@ -44,13 +44,13 @@
     "zone.js": "^0.8.26"
   },
   "peerDependencies": {
-    "@angular-devkit/core": "^0.6.1",
-    "@angular/common": ">=6.0.0",
-    "@angular/compiler": ">=6.0.0",
-    "@angular/core": ">=6.0.0",
-    "@angular/forms": ">=6.0.0",
-    "@angular/platform-browser": ">=6.0.0",
-    "@angular/platform-browser-dynamic": ">=6.0.0",
+    "@angular-devkit/core": "^0.6.1 || >=7.0.0",
+    "@angular/common": ">=6.0.0 || >=7.0.0",
```

Comment:
Doesn't `>=6.0.0` range include `>=7.0.0` as well ? IMO we  need to fix only `@angular-devkit/core`

Reply:
Good point! Done